extensions/aws-eks/runtime/src/main/resources/META-INF/quarkus-extension.json

```diff
@@ -0,0 +1,10 @@
+{
+  "name": "Camel Quarkus AWS EKS",
+  "labels": [
+    "integration",
+    "camel",
+    "aws",
+    "eks"
```

Comment:
Do we need those sub labels? Wont aws be sufficient? And eg if you search for eks then I assume it can also search in the name? And maybe we should add cloud as a label for all those aws extensions?